recipes/pygmsh/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "pygmsh" %}
+{% set version = "7.1.17" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/pygmsh-{{ version }}.tar.gz
+  sha256: 563724abff4f8517598d3354b9c795feddbe0675786145bfeaf20cbee6691c1d
+
+build:
+  entry_points:
+    - pygmsh-optimize = pygmsh._cli:optimize_cli
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python
+    - flit-core >=3.2,<4
+    - pip
+    - libgl-devel  # [linux]
```

Comment:
The `pygmsh` package is basically `noarch` but `gmsh` requires `libgl` so the package is no longer `noarch`.